Makefile

```diff
@@ -43,11 +47,23 @@ build-debian: debian/base/include/etc/confluent/docker/docker-utils.jar
 		else \
 			BUILD_ARGS=""; \
 		fi; \
-		docker build --build-arg COMMIT_ID=${COMMIT_ID} --build-arg BUILD_NUMBER=${BUILD_NUMBER} $${BUILD_ARGS} -t confluentinc/cp-$${component}:latest debian/$${component} || exit 1 ; \
-		docker tag confluentinc/cp-$${component}:latest ${REPOSITORY}/cp-$${component}:latest  || exit 1 ; \
-		docker tag confluentinc/cp-$${component}:latest ${REPOSITORY}/cp-$${component}:${CP_VERSION} || exit 1 ; \
-		docker tag confluentinc/cp-$${component}:latest ${REPOSITORY}/cp-$${component}:${VERSION} || exit 1 ; \
-		docker tag confluentinc/cp-$${component}:latest ${REPOSITORY}/cp-$${component}:${COMMIT_ID} || exit 1 ; \
+		for type in "" rpm; do \
+			DOCKER_FILE="debian/$${component}/Dockerfile"; \
+			COMPONENT_NAME=$${component}; \
+			echo $${type}; \
+			if [ "$${type}" = "rpm" ]; then \
+				echo "this an rpm"; \
+				COMPONENT_NAME="rpm-$${component}"; \
+				DOCKER_FILE="$${DOCKER_FILE}.rpm"; \
+			fi; \
+			if [ -a "$${DOCKER_FILE}" ]; then \
+				docker build --build-arg COMMIT_ID=${COMMIT_ID} --build-arg BUILD_NUMBER=${BUILD_NUMBER} $${BUILD_ARGS} -t confluentinc/cp-$${COMPONENT_NAME}:latest -f $${DOCKER_FILE} debian/$${component} || exit 1 ; \
+				docker tag confluentinc/cp-$${COMPONENT_NAME}:latest ${REPOSITORY}/cp-$${COMPONENT_NAME}:latest  || exit 1 ; \
```

Comment:
How come we use "confluentinc" in the source image while using $REPOSITORY in the target image even though they are the same value -- this is no op? Same with the build command and other tags below where 'confluentinc' is used instead of $REPOSITORY.